app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+
+#### Example Workspace Migration
+
+The following provides an example set of steps for migrating a small multi-Workspace setup to 
+{{site.konnect_product_name}}.
+
+{:.note}
+> **Note**: Instructions on this page are not step-by-step guides. They are meant to be illustrative of the general steps 
+you can perform to migrate Workspaces. The examples use [decK](/deck/latest/) and some well known command line tools for 
+querying APIs and manipulating text output. See the [jq](https://jqlang.github.io/jq/) and [yq](https://github.com/mikefarah/yq) 
+tool pages for more information.
+
+We can start by querying the [Admin API](/gateway/latest/admin-api/) of our on-premises installation to get 
+a list of workspaces for a particular {{site.base_gateway}} installation:
+
+```sh
+curl -s localhost:8001/workspaces | jq -r '.data.[].name'
+```
+
+And we'll receive a response with a name for each Workspace:
+
+```text
+default
+inventory
+sales
+```
+
+Our example on-premises installation has three total workspaces: The `default` workspace, `inventory`, and `sales`. 
+
+{:.note}
+> **Note**: {{site.base_gateway}} provides a `default` Workspace, and similarly {{site.konnect_product_name}} provides a 
+`default` Control Plane. Neither of these can be deleted, so migrating the `default` Workspace to the `default` Control Plane 
+is a logical choice.
+
+Create a new Control Plane for each non-default Workspace:
+
+```txt
+curl -s -H "Authorization: Bearer ${KONNECT_PAT}" \
+  --request POST \
+  --url https://us.api.konghq.com/v2/control-planes \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{"name":"inventory","description":"CP for the Inventory team.","cluster_type":"CLUSTER_TYPE_CONTROL_PLANE"}'
+```
+
+```txt
+curl -s -H "Authorization: Bearer ${KONNECT_PAT}" \
+  --request POST \
+  --url https://us.api.konghq.com/v2/control-planes \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{"name":"sales","description":"CP for the Sales team.","cluster_type":"CLUSTER_TYPE_CONTROL_PLANE"}'
+```
+
+Login into the Konnect UI and validate the new Control Planes.
+
+### Plugins
+
+Konnect supports the majority of plugins available to {{site.base_gateway}}, however,
+{{site.base_gateway}} is ran in Hybrid Mode which limits support for plugins that require direct access
+to a database. 
+
+{:.note}
+> **Note**: Konnect also provides Dedicated Cloud Gateways which 
+further limit plugins that require specialized software agents running on the dataplane hosts. 
+
+In order to migrate plugins from on-premises to Konnect you should review 
+[Konnect Compatibility page](/konnect/compatibility/) for your usage of unsupported
+plugins. Additionally, review if certain configuration values are unsupported which will require additional
+changes to your configuration.
```

Comment:
```suggestion
{{site.konnect_short_name}} supports the majority of plugins available to {{site.base_gateway}}. Since {{site.konnect_short_name}} runs in hybrid mode, this limits support for plugins that require direct access
to a database. 

{:.note}
> **Note**: {{site.konnect_short_name}} also provides Dedicated Cloud Gateways, which 
further limit plugins that require specialized software agents running on the data plane hosts. 

To migrate plugins from a self-managed deployment to {{site.konnect_short_name}}, review 
[Konnect Compatibility page](/konnect/compatibility/) to check for supported and unsupported plugins.
Also review any plugin configuration values, as certain values are unsupported in {{site.konnect_short_name}} and may require additional
changes to your configuration.
```